packages/@aws-cdk/aws-ec2/lib/vpc-flow-logs.ts

```diff
@@ -257,13 +257,6 @@ class CloudWatchLogsDestination extends FlowLogDestination {
  * @experimental
  */
 export interface FlowLogOptions {
-  /**
-   * The options for creating a CloudWatch log group
-   *
-   * @default - will use logs.LogGroupProps defaults
-   */
-  readonly logGroupOptions?: logs.LogGroupProps;
```

Comment:
You should not have been allowed to remove this property, because it breaks backwards compatibility.

Reply:
Aha the API is experimental of course.